### ISSUE TITLE
Membership slider view doesn't work as expected

### ISSUE THREAD

**Issue by shadeed** [smashingmagazine/redesign#147]
**- Do you want to request a *feature* or report a *bug*?**
Bug.

**- What is the current behavior?**
When moving the slider handle, the input range green bar goes back to zero. (Attached a GIF)

**- If the current behavior is a bug, please provide the steps to reproduce.**
Check on iOS Opera Mini

![smashing-slider](https://cloud.githubusercontent.com/assets/8038659/24111126/1a86bfec-0d9e-11e7-9de4-f65383689096.gif)

**- What is the expected behavior?**
Go to https://next.smashingmagazine.com/member/

**- Please mention your operating system version and the version of your browser**
Opera Mini (14.0.0.104835), iOS10.

**Comment by vitalyfriedman** [comment]
Good point @shadeed — datalist isn't supported in Opera Mini, so we'll have to provide a meaningful fallback. Thanks for reporting the issue!
